components/prize_pool/wikis/counterstrike/prize_pool_custom.lua

```diff
@@ -0,0 +1,115 @@
+---
+-- @Liquipedia
+-- wiki=counterstrike
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Opponent = require('Module:Opponent')
+local Namespace = require('Module:Namespace')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+
+local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector', {requireDevIfEnabled = true})
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+local TIER_VALUE = {10, 6, 4, 2}
+local TYPE_MODIFIER = {offline = 1, ['offline/online'] = 0.75, ['online/offline'] = 0.75, default = 0.65}
+
+local HEADER_DATA = {}
+
+-- Template entry point
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+
+	-- Turn off automations
+	args.prizesummary = false
+	args.autousd = false
+	args.exchangeInfo = false
+
+	local prizePool = PrizePool(args):create()
+
+	prizePool:setLpdbInjector(CustomLpdbInjector())
+
+	if args['smw mute'] or not Namespace.isMain() then
```

Comment:
also check for `disable_SMW_storage` wiki var?
(it could get set by a template or manually before the PP)